buildkite/scripts/rosetta-integration-tests.sh

```diff
@@ -52,12 +55,16 @@ echo "=========================== GENERATING GENESIS LEDGER FOR ${MINA_NETWORK}
 mkdir -p $MINA_KEYS_PATH
 mina-dev advanced generate-keypair --privkey-path $MINA_KEYS_PATH/block-producer.key
 mina-dev advanced generate-keypair --privkey-path $MINA_KEYS_PATH/snark-producer.key
+mina-dev advanced generate-keypair --privkey-path $MINA_KEYS_PATH/faucet.key
 chmod -R 0700 $MINA_KEYS_PATH
 BLOCK_PRODUCER_PK=$(cat $MINA_KEYS_PATH/block-producer.key.pub)
 SNARK_PRODUCER_PK=$(cat $MINA_KEYS_PATH/snark-producer.key.pub)
+FAUCET_PK=$(cat $MINA_KEYS_PATH/faucet.key.pub)
+FAUCET_SK=$(mina-dev advanced dump-keypair --privkey-path $MINA_KEYS_PATH/faucet.key | grep "Private key" | sed -e "s/Private key: //")
```

Comment:
The use of `grep` here can be replaced with additional options to `sed` like this:
```suggestion
FAUCET_SK="$(mina-dev advanced dump-keypair --privkey-path $MINA_KEYS_PATH/faucet.key | sed -ne "s/Private key: //p")"
```

Reply:
Indeed. This line was deleted, but I'm now using `sed` on similar commands.